waku/v2/protocol/waku_message.nim

```diff
@@ -12,8 +12,15 @@ import
   libp2p/protobuf/minprotobuf,
   libp2p/varint,
   ../utils/protobuf,
-  ../utils/time,
-  waku_rln_relay/waku_rln_relay_types
+  ../utils/time
+
+## TODO: properly address these import once zerokit rln is merged
+when defined(rln) or (not defined(rln) and not defined(rlnzerokit)):
+  import waku_rln_relay/waku_rln_relay_types
+
+when defined(rlnzerokit):
+  import waku_rln_relay/rlnzerokit/waku_rln_relay_types
+
```

Comment:
```suggestion
```suggestion
## TODO: properly address these import once zerokit rln is merged
when defined(rln):
  import ../protocol/waku_rln_relay/waku_rln_relay_types
else:
  when defined(rlnzerokit):
     import ../protocol/waku_rln_relay/rlnzerokit/waku_rln_relay_types
```

```

Reply:
Such separation is not needed when the same file is used.

Reply:
Ditto, Once we remove duplicate waku-rln-relay-types.nim files, then there is no need for conditional import.